src/Utils.hh

```diff
@@ -24,9 +24,56 @@
 #ifndef SRC_UTILS_HH
 #define SRC_UTILS_HH
 
+#include "common.hh"
+
+#include "hip/hip_fatbin.h"
+
+#include <filesystem>
+#include <optional>
```

Comment:
unnecessary if included in common.hh

Reply:
Filesystem include is gone. Didn't see optional included in common.h.